### ISSUE TITLE
the proxy is down

### ISSUE THREAD

**Issue by rickroll2222** [PrismarineJS/prismarine-web-client#371]

**Comment by zardoy** [comment]
@rickroll2222 Indeed, but you can use many other community proxies or try my proxy: proxy.mcraft.fun which has 100% uptime [ref](https://github.com/zardoy/minecraft-web-client/issues/3#issuecomment-2010987530) and supports authentication ;)

**Comment by rom1504** [comment]
Yes my server is down. I'll fix it in next days

**Comment by zardoy** [comment]
> I'll fix it in next days

Please make sure it's updated with the latest net-browserify, thx!
